agent/init/business/business.go

```diff
@@ -38,3 +39,9 @@ func syncSSL() {
 		global.LOG.Errorf("sync ssl status error : %s", err.Error())
 	}
 }
+
+func syncTask() {
+	if err := service.NewITaskService().SyncForRestart(); err != nil {
+		global.LOG.Errorf("sync task status error : %s", err.Error())
+	}
+}
```

Comment:
No Issues Found. The provided code has no obvious issues that require manual fixes for consistency, clarity or efficiency. No suggestions were offered to optimize this specific function body or its overall structure as there appear to be minimal structural changes made since the knowledge cutoff of 2021-09-01. If you have more recent codebase, feel free to share it so I could potentially analyze it further.